image-index.md

```diff
@@ -55,19 +55,29 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`os.version`** *string*
 
-        This OPTIONAL property specifies the operating system version, for example `10.0.10586`.
+        This OPTIONAL property specifies the operating system version.
+        Image indexes SHOULD use, and implementations SHOULD understand the following convention:
+        - `windows`: `sprintf("%d.%d.%d", dwMajorVersion, dwMinorVersion, dwBuildNumber)` with the fields of the `OSVERSIONINFOEX` structure returned by `GetVersionEx()`. e.g., `10.0.10586`.
+        If an operating system is not listed here, it SHOULD be submitted to this specification for standardization.
 
     - **`os.features`** *array of strings*
 
         This OPTIONAL property specifies an array of strings, each specifying a mandatory OS feature (for example on Windows `win32k`).
+        A convention for other features SHOULD be submitted to this specification for standardization.
 
     - **`variant`** *string*
 
-        This OPTIONAL property specifies the variant of the CPU, for example `armv6l` to specify a particular CPU variant of the ARM CPU.
+        This OPTIONAL property specifies the variant of the CPU.
+        Image indexes SHOULD use, and implementations SHOULD understand the following convention:
+        - `arm` and `arm64`: `machine` field of the Linux `utsname` structure returned by `uname(2)`. e.g., `armv6l`.
+        If an architecture is not listed here, it SHOULD be submitted to this specification for standardization.
```

Comment:
How about marking this property "reserver, SHOULD NOT use" if there is a concern?
I think we don't need to rush to standardize this property toward v1.0.0